exo-add/features/exoservice-go.feature

```diff
@@ -0,0 +1,79 @@
+Feature: scaffolding an ExoService written in ES6
+
+  As a JavaScript developer adding features to an Exosphere application
+  I want to have an easy way to scaffold an empty ExoService
+  So that I don't waste time copy-and-pasting a bunch of code.
+
+  - run "exo add service <name> exoservice-es6"
+    to add a new exo-service written in ES6 to the current application
+  - run "exo add service" to add the service interactively
+
+
+  Scenario: calling with all command line arguments
+    Given I am in the root directory of an empty application called "test app"
+    When running "exo-add service --service-role=users --service-type=users --author=test-author --template-name=exoservice-go --model=user --description=testing" in this application's directory
+    Then my application contains the file "application.yml" with the content:
+      """
+      name: test app
+      description: Empty test application
+      version: 1.0.0
+
+      bus:
+        type: exocom
+        version: 0.21.7
+
+      services:
+        public:
+          users:
+            location: ./users
+      """
+    And my application contains the file "users/service.yml" with the content:
+      """
+      type: users
+      description: testing
+      author: test-author
+
+      setup: glide install
+      startup:
+        command: go run server.go
+        online-text: online at port
+      tests: node_modules/cucumber/bin/cucumber.js
+
+      messages:
+        receives:
+          - ping
+        sends:
+          - pong
+
+      dependencies:
+      """
+    And my application contains the file "users/server.go" with the content:
+      """
+      package users
+
+      import (
+      	"fmt"
+
+      	"github.com/Originate/exocom/go/exorelay"
+      	"github.com/Originate/exocom/go/exoservice"
+      )
+
+      func handlePing(request exoservice.Request) {
+      	err := request.Reply(exorelay.MessageOptions{Name: "pong"})
```

Comment:
We have `structs.Message` on exocom that is the for what is passed on the wire. `MessageOptions` is an object with only Name / Payload / ResponseTo. It could be renamed to `exorelay.Message` if you think that makes sense. I think that would be a bit confusing with `structs.Message`.

We could also move rename `structs.Message` -> `structs.RawMessage` and `exorelay.MessageOptions` -> `structs.Message`